recipes-firmware/firmware/firmware-qcom-boot-common.inc

```diff
@@ -0,0 +1,16 @@
+# Install NHLOS boot binaries in DEPLOY_DIR
+
```

Comment:
Please add:
```
UNPACKDIR = "${S}"
```

Fixes:
```
WARNING: firmware-qcom-boot-qcm6490-00005.0-r0 do_unpack: firmware-qcom-boot-qcm6490: the directory ${WORKDIR}/${BP} (/home/rsalveti/build/quic-yocto/build/tmp/work/armv8-2a-poky-linux/firmware-qcom-boot-qcm6490/00005.0/firmware-qcom-boot-qcm6490-00005.0) pointed to by the S variable doesn't exist - please set S within the recipe to point to where the source has been unpacked to
```

Basically UNPACKDIR is not set by default to the expected path from S here.